feat(project landing page): add copy to clipboard functionality (DSP-1248)

waychal opened the pull request:
resolves DSP-1248

adds copy to clipboard functionality for `citation link`, `persistent identifier` and `license` using Angular CDK clipboard module.

mpro7 reviewed (commented):
It seems good, but before you merge:
1. Have a look on my [late comments](https://github.com/dasch-swiss/dsp-app/pull/366#pullrequestreview-573295765) from last PR.
2. DSP-1284 doesn't exists.
3. Don't put space between PR title type and scope `feat():` instead of `feat ():`.
4. Creating branch start with `wip/` prefix, so `wip/dsp-xxx` instead of just 'dsp-xxx`.

waychal commented:
@mpro7

> 1. Have a look on my [late comments](https://github.com/dasch-swiss/dsp-app/pull/366#pullrequestreview-573295765) from last PR.

I have updated this PR for your comments from another PR.

> 2. [DSP-1284](https://dasch.myjetbrains.com/youtrack/issue/DSP-1284) doesn't exists.

Thanks, I have corrected it in PR title and description.

> 3. Don't put space between PR title type and scope `feat():` instead of `feat ():`.

Done.

> 4. Creating branch start with `wip/` prefix, so `wip/dsp-xxx` instead of just `dsp-xxx`.

Normally when I create PR they are no longer work in progress (wip). Hence I do not add `wip/` in branch name.
If I still have to use the prefix `wip/` for branch name even if it is ready for review, let me know and I will do it from next time. For this one, I will keep it as it is.

mpro7 commented:
> @mpro7
> 
> > 1. Have a look on my [late comments](https://github.com/dasch-swiss/dsp-app/pull/366#pullrequestreview-573295765) from last PR.
> 
> I have updated this PR for your comments from another PR.
> 
> > 1. [DSP-1284](https://dasch.myjetbrains.com/youtrack/issue/DSP-1284) doesn't exists.
> 
> Thanks, I have corrected it in PR title and description.
> 
> > 1. Don't put space between PR title type and scope `feat():` instead of `feat ():`.
> 
> Done.
> 
> > 1. Creating branch start with `wip/` prefix, so `wip/dsp-xxx` instead of just `dsp-xxx`.
> 
> Normally when I create PR they are no longer work in progress (wip). Hence I do not add `wip/` in branch name.
> If I still have to use the prefix `wip/` for branch name even if it is ready for review, let me know and I will do it from next time. For this one, I will keep it as it is.

Thanks!
This what we have in [our contribution guideline](https://docs.dasch.swiss/developers/dsp/contribution/#create-branch-guidelines) and I found it useful while working with git on terminal :).
